src/tecton_mcp/scripts/generate_embeddings.py

```diff
@@ -8,9 +8,47 @@
 import tecton
 import pandas as pd
 
+# ---------------------------------------------------------------------------
+# Configuration for each documentation version we want to process. The output
+# database filename MUST match the resolution logic in
+# `documentation_tools._resolve_docs_db_path()`.
+# ---------------------------------------------------------------------------
+
+
 EMBED_MODEL = "sentence-transformers/multi-qa-MiniLM-L6-cos-v1"
-DOCS_PATH = os.path.expanduser("~/git/tecton-docs/docs")
-DOCS_BASE_URL = "https://docs.tecton.ai/docs/beta/"
+
+DOC_VERSIONS = [
+    {
+        "name": "latest",  # corresponds to >1.1.x (default / beta)
+        "db_filename": "tecton_docs.db",
+        "docs_path": os.path.expanduser("~/git/tecton-docs/docs"),
+        "base_url": "https://docs.tecton.ai/docs/beta/",
+    },
+    {
+        "name": "1.1",
+        "db_filename": "tecton_docs_1.1.db",
+        "docs_path": os.path.expanduser("~/git/tecton-docs/versioned_docs/version-1.1"),
+        "base_url": "https://docs.tecton.ai/docs/",
+    },
+    {
+        "name": "1.0",
+        "db_filename": "tecton_docs_1.0.db",
+        "docs_path": os.path.expanduser("~/git/tecton-docs/versioned_docs/version-1.0"),
+        "base_url": "https://docs.tecton.ai/docs/1.0",
+    },
+    {
+        "name": "0.9",
```

Comment:
Let’s drop 0.9 and 0.8 - we’re only supporting 1.0 and above with the MCP server